package.json

```
@@ -45,6 +45,14 @@
     "nock": "^10.0.1",
     "standard-version": "^4.2.0"
   },
+  "overrides": {
+    "joi": {
```

Does this work? 
```suggestion
    "joi@13.7.0": {
```
If this works, it would be safer to pin to joi v13 to avoid the risk of overriding other versions of joi when they're present.

---

@dhei : Current joi version in npms-queries is `"joi": "^14.0.2"`? Any reason to pin at 13.7.0 ? I don't know the consequences of downgrading.

---

Sorry for the confusion, I have the wrong assumption that npms-queries is using the same joi version as ormnomnom does.

l think it makes sense to ping to 14.0.2. In my testing I found that `overrides` didn't seem to work well with `^` in semver, so you might need pin to an exact version like `joi@14.0.2`.